documents/virtualAccount.md

```diff
@@ -292,7 +292,7 @@ String virtualId = "va_DlGmm7jInLudH9";
 JSONObject params = new JSONObject();
 params.put("count","1");
         
-List<VirtualAccount> virtualaccounts = instance.virtualAccounts.fetchPayments(virtualId,params);
+List<Payment> payments = razorpayclient.virtualAccounts.fetchPayments(virtualId,params);
```

Comment:
package razorpayclient does not exist

Reply:
[fixed](https://github.com/razorpay/razorpay-java/blob/smartcollect_doc_collection/documents/virtualAccount.md?plain=1#L295)